arith/gf2_128/src/tests.rs

```diff
@@ -13,7 +13,8 @@ use crate::{GF2_128x8, GF2_128};
 
 #[test]
 fn test_simd_field() {
-    random_simd_field_tests::<GF2_128x8>("Simd GF2 Ext128".to_string());
+    random_simd_field_tests::<GF2_128>("Simd for GF2 over GF2Ext128".to_string());
```

Comment:
The same SIMD field problem here.

Reply:
reverted in https://github.com/PolyhedraZK/Expander/pull/142/commits/4fd26abc44b17fa828b34d4bccf3a2eba86029d4